src/main/scala/uk/gov/nationalarchives/Main.scala

```diff
@@ -49,6 +52,16 @@ object Main extends IOApp {
   override def run(args: List[String]): IO[ExitCode] =
     for {
       config <- ConfigSource.default.loadF[IO, Config]()
+      _ <-
+        Stream
+          .emits {
+            val versionWithoutSnapshot: String = BuildInfo.version.stripSuffix("-SNAPSHOT")
+            versionWithoutSnapshot.getBytes("UTF-8")
+          }
+          .through(Files[IO].writeAll(Path(s"${config.versionPath}-${UUID.randomUUID()}")))
```

Comment:
You need a forward slash here. The config path will be something like `/tmp/versions` and the file will be `/tmp/versions/eb75effa-bd97-4312-949d-9eec179fa1e3` so this needs to be `s"${config.versionPath}/${UUID.randomUUID()}"`